<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>contextual_replacement</class>
 <widget class="QDialog" name="contextual_replacement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="end_line_val">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>183</y>
     <width>175</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="first_val">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>35</y>
     <width>175</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Insert</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>193</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>New substring:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="start_line_val">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>133</y>
     <width>175</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>First line:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Substring:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="last_val">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>85</y>
     <width>175</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Last line:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>contextual_replacement</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
